diff --git a/src/components/BaseHead.astro b/src/components/BaseHead.astro
index ca1e1ee..cbda22e 100644
--- a/src/components/BaseHead.astro
+++ b/src/components/BaseHead.astro
@@ -1,6 +1,7 @@
 ---
 import '../styles/global.css';
 import { SITE_TITLE } from '../consts.ts';
+import { Font } from 'astro:assets';
 interface Props {
   title: string;
   description: string;
@@ -11,6 +12,8 @@ const canonicalURL = new URL(Astro.url.pathname, Astro.site);
 const { title, description, image = '/blog-placeholder-1.jpg' } = Astro.props;
 ---
 
+<Font cssVariable="--font-atkinson" preload />
+
 <!-- Favicon & Sitemap -->
 <link rel="icon" type="image/svg+xml" href="/favicon.svg?v=3" />
 <link rel="icon" href="/favicon.ico" />
